nautobot/ipam/templates/ipam/rir_list.html

```diff
@@ -1,23 +0,0 @@
-{% extends 'generic/object_list.html' %}
```

Comment:
What's this deletion for?

Reply:
Removing unused/unneeded UI cruft - all this file did was add an "IPv4 Stats"/"IPv6 Stats" toggle button that never (in Nautobot, at least) ever actually worked - try it out at https://demo.nautobot.com/ipam/rirs/ for yourself :)

Reply:
🤦 Good catch!